### PR TITLE
RK-322 add tagging graphics

### PR DESCRIPTION
**Pull request by Rookeret73**

### CODE REVIEW

File: docs/workspaces-tagging.md [hunk 1]
```diff
@@ -4,49 +4,87 @@ title: Tagging
...
-1. When installing the Rookout SDK into your code, provide the environment name as a tag to the Rookout SDK.  
+### Debug all instances of Service A
+
+You may use tag:ServiceA to debug all instances of Service A (both production and staging).
+
+1. When installing the Rookout SDK into your code, provide the service name as a tag to the Rookout SDK.  
 For example, when using the Rookout SDK start() API, pass the following variable:
 ```python
-rook.start(token=[Your Rookout Token], tags=["Staging"])
+rook.start(token=[Your Rookout Token], tags=["ServiceA"])
```
[the remark this answers is not in the document]
Comment (Rookeret73) [reply docs/workspaces-tagging.md]: Done.

File: docs/workspaces-tagging.md [hunk 2]
```diff
@@ -4,49 +4,87 @@ title: Tagging
...
 ```
 2. When creating your Rookout Workspace, use the following tag:
```
[the remark this answers is not in the document]
Comment (Rookeret73) [reply docs/workspaces-tagging.md]: I did :)
Will change the formatting a bit so it is easier to see.

**Comment by Rookeret73** [comment]
Applied fixes and pushed.

**Comment by Rookeret73** [comment]
Added examples for the system configuration and pushed again.
Please take a look.

File: docs/workspaces-tagging.md [hunk 3]
```diff
@@ -4,49 +4,72 @@ title: Tagging
...
+For example: `hostname:demo-deployment`  
+
+ip - the ip address of an application instance.  
+For example: `ip:25.218.30.111`  
+
+platform - the platform used an application instance.  
+For example: `platform:python`  
 
+processname - the process name (executable) of an application instance.  
+For example: `processname:demo/app.py`
```
Comment (LiranLast) [inline docs/workspaces-tagging.md]: @Rookeret73 I'm not sure that's a good value example. Did you check?

File: docs/workspaces-tagging.md [hunk 4]
```diff
@@ -4,49 +4,72 @@ title: Tagging
...
+In addition to Tags, you may filter using system configuration identified by the Rookout SDK:  
+
+hostname -   the host name of an application instance.  
+For example: `hostname:demo-deployment`  
+
+ip - the ip address of an application instance.  
+For example: `ip:25.218.30.111`  
+
+platform - the platform used an application instance.  
+For example: `platform:python`
```
Comment (LiranLast) [inline docs/workspaces-tagging.md]: Let's list all possible values here- it's not trivial

File: docs/workspaces-tagging.md (hunk 3 above)
Comment (LiranLast) [reply docs/workspaces-tagging.md]: +1

File: docs/workspaces-tagging.md (hunk 4 above)
Comment (LiranLast) [reply docs/workspaces-tagging.md]: +1

**Comment by Rookeret73** [comment]
Added supported platforms and mentioned that the user can find values for his own app instances in the App Instances page.

**Pull request merged**